importers/degree-importer.php

```diff
@@ -688,6 +692,12 @@ private function process_post_terms( $post_id, $post_terms ) {
 				}
 
 				if ( $term_id ) {
+					// Set the alias
+					$alias = $this->get_college_alias( $term );
+					if ( ! add_term_meta( $term_id, 'colleges_alias', $alias, true ) ) {
+						update_term_meta( $term_id, 'colleges_alias', $alias, true );
```

Comment:
In this case, I probably don't need to do the additional check... also the `true` variable in the 4th position is incorrect for `update_term_meta`. The 4th parameter is supposed to be `$prev_value` which I'm not going to be using.